src/snowflake/snowpark/mock/_telemetry.py

```diff
@@ -83,9 +87,8 @@ def _upload_payload(self, payload) -> None:
         success = True
         response = None
         try:
-            # import here is because stored proc doesn't have vendored request module
-            # have it at the top will cause import error in stored procedure running
-            from snowflake.connector.vendored import requests
+            # import here is because stored proc doesn't have requests as requirement
+            import requests
```

Comment:
Question: Why do we use non vendored requests here?

Reply:
the vendored requests triggers unnecessary ocsp validation